### ISSUE TITLE
NTR: C14:1: tetradecenoylcarnitine high during metabolic stress in blood (Lab: chemistry: carnitine compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8562]
New term request C14:1: tetradecenoylcarnitine high during metabolic stress in blood (segal_030514184200): Acylcarnitines

**Comment by pnrobinson** [comment]
@MickeySegal
analogous question as for https://github.com/obophenotype/human-phenotype-ontology/issues/8561

**Comment by MickeySegal** [comment]
This is like https://hpo.jax.org/app/browse/term/HP:0012726.  The chemistry finding is only present episodically.

**Comment by pnrobinson** [comment]
@MickeySegal 

I think it is better not to add metabolic stress to the term, because lots of abnormalities are made worse by various kinds of metabolic stress. I would suggest to combine with a "[Triggered by](https://hpo.jax.org/app/browse/term/HP:0025204)" modifier. There is not currently a "Triggered by metabolic stress", but I would think that it might be better to describe more precisely what the stress factor is.

This is the term for the "elevated"

Adding new term: 
- ID: HP:6000476
- label: Elevated circulating tetradecenoylcarnitine concentration
- parents: [Elevated circulating tetradecanoylcarnitine concentration (HP:0033465)](https://hpo.jax.org/app/browse/term/HP:0033465)
- pmid: 
- def: The concentration of tetradecenoylcarnitine (C14:1) in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8562
- synonyms:
